build.gradle

```diff
@@ -152,7 +152,8 @@ project.ext.externalDependency = [
     'testContainersCassandra': 'org.testcontainers:cassandra:1.15.3',
     'typesafeConfig':'com.typesafe:config:1.4.1',
     'wiremock':'com.github.tomakehurst:wiremock:2.10.0',
-    'zookeeper': 'org.apache.zookeeper:zookeeper:3.4.14'
+    'zookeeper': 'org.apache.zookeeper:zookeeper:3.4.14',
+    'log4j': 'org.apache.logging.log4j:log4j-1.2-api:2.17.1'
```

Comment:
why do we need to pull this one in?

Reply:
It is required for Apache Ranger library, I removed this from here and added in the auth-ranger-impl module